README.md

```diff
@@ -47,113 +47,163 @@ If you find `googler` useful, please consider donating via PayPal.
 
 `googler` requires Python 2.7.x or Python 3.x to work.
 
-1. If you have git installed (the steps are tested on Ubuntu 14.04.3 x64_64):  
-<pre>$ git clone https://github.com/jarun/googler/  
-$ cd googler  
-$ sudo make install</pre>  
-To remove, run:  
-<pre>$ sudo make uninstall</pre>
+## Installing from this repository
+
+If you have git installed:
+
+    $ git clone https://github.com/jarun/googler/
+    $ cd googler
+
+Otherwise, download the latest [stable release](https://github.com/jarun/googler/releases/latest) or [development version](https://github.com/jarun/googler/archive/master.zip"), unpack the archive, and cd into the directory.
+
+Next, run
+
+    $ make install
+
+which installs to `/usr/local` by default. You might need to prepend `sudo` to the command above if `/usr/local` is only writable by root. To install to a different prefix, run
+
+    $ PREFIX=/path/to/prefix make install
+
+To remove, run
+
+    $ make uninstall
+
+Again, you might need `sudo` and/or `PREFIX=`.
+
+### Don't feel like installing to a prefix?
```

Comment:
Switched to "Running as a standalone utility" to be consistent with other section/subsection names, because we were using nouns or present participles throughout.